x-pack/plugins/apm/server/lib/traces/get_trace_errors_per_transaction.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SearchParams } from 'elasticsearch';
+import {
+  PROCESSOR_EVENT,
+  TRACE_ID
+} from 'x-pack/plugins/apm/common/elasticsearch_fieldnames';
+import { Setup } from '../helpers/setup_request';
+
+export interface ErrorsPerTransaction {
+  [transactionId: string]: number;
+}
+
+interface TraceErrorsAggBucket {
+  key: string;
+  errors: {
+    doc_count: number;
+  };
+}
+
+interface TraceErrorsAggResponse {
+  transactions: {
+    buckets: TraceErrorsAggBucket[];
+  };
+}
+
+export async function getTraceErrorsPerTransaction(
+  traceId: string,
+  setup: Setup
+): Promise<ErrorsPerTransaction> {
+  const { start, end, client, config } = setup;
+
+  const params: SearchParams = {
+    index: [config.get('apm_oss.errorIndices')],
```

Comment:
Since this is querying the error index, should this file perhaps be located under "lib/errors"? 
Not necessarily saying it should, so trying to find a system that makes sense.